### ISSUE TITLE
Insufficient color contrast - list of elements to change

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#2593]
For more details on issue check: https://github.com/TryQuiet/quiet/issues/2592

Some elements had insufficient color contrast and Jason provided informations necessary to fixe it:

Blue links: Yes, this is deprecated. We have a darker blue (#1B6FEC) in our global Figma variables/tokens that we use for inline links such as these. It has a contrast of 4.63. Let's use this instead.
Placeholder text in forms: Perfect, I didn't catch this one, let's use #767676 for the placeholder text which is 4.54. I've added this to the global variables/tokens in Figma and set the placeholder text to use it in all forms.

Colors need to be replaced in below places (mobile and desktop):
- placeholders in "Join community", "Create your community" and in general area users use to write a message,
- links "create a new community", "join a community" 
- link that is send by users in message

Below you will find print screens from mentioned areas:
<img width="592" alt="Screenshot 2024-08-07 at 14 10 30" src="https://github.com/user-attachments/assets/8b0fd843-e336-4307-9ef3-0f21717383a6">
<img width="627" alt="Screenshot 2024-08-07 at 14 11 03" src="https://github.com/user-attachments/assets/753b3b66-3385-4de9-b632-dc8e22d506d1">
<img width="203" alt="Screenshot 2024-08-07 at 14 15 26" src="https://github.com/user-attachments/assets/902c0c9a-1eb6-4ee7-a5dc-6d036f1aa2c4">

**Comment by kingalg** [comment]
New and old color contrasts added in PR.

<img width="1026" alt="Screenshot 2024-10-15 at 14 30 46" src="https://github.com/user-attachments/assets/e76f4202-1bed-454d-b92a-4915c93c5792">

<img width="948" alt="Screenshot 2024-10-16 at 11 22 17" src="https://github.com/user-attachments/assets/9aaebf63-aa80-4d72-91e2-d8db70b16e77">

**Comment by kingalg** [comment]
Applied in version - 2.3.3-alpha.1 - checked and released.
